### ISSUE TITLE
JS error in console - after update Contao 4.9.1 to 4.9.2

### ISSUE THREAD

**Issue by planepix** [terminal42/contao-rte_table#5]
Hi guys, 
i want to leave this message / error message:

After updating Contao 4.9.1 to 4.9.2 i got a blank white screen when i want to edit a table cell.

Got this error in console:
```
Uncaught TypeError: Cannot read property 'set' of null at frm.onload (table-wizard.js:28)
```
Line 28: ``` frm.document.getElementById('rte-table-editor').set('value', content);```

Version 1.0.4 was installed. Also tried to install the dev-master version which makes no differences.

Thanx a bunch

**Comment by qzminski** [comment]
I am not able to reproduce that issue on Contao 4.9.2 🤔 Is it a local or remote website? Which browsers are affected? Did you clear the browser cache?

**Comment by planepix** [comment]
Hi Kamil,
it happens - in Chrome also as in Firefox - after clearing the Cache (Contao and Symfony).
This is a remote site, protected by htaccess.

I add a few screenshots - otherwise i can give you access to the installation by mail.

![rt1-1](https://user-images.githubusercontent.com/1823889/78420552-76c05980-7650-11ea-9894-e8b3dcff0dd6.png)
![rte-2](https://user-images.githubusercontent.com/1823889/78420558-7d4ed100-7650-11ea-9368-06aee2e43bf4.png)
![rt1-3](https://user-images.githubusercontent.com/1823889/78420562-82ac1b80-7650-11ea-8079-d0f13417a2e5.png)

**Comment by planepix** [comment]
Tested also in two other sites: all the same results as described above.

**Comment by qzminski** [comment]
Would you be able to provide me the access details to any of those systems? I will need an access to Contao backend and FTP server. My e-mail is: kamil.kuzminski@codefog.pl

**Comment by planepix** [comment]
Send you the details. Thank you.

**Comment by qzminski** [comment]
It should be fixed in 1.0.5 👍🏻

**Comment by planepix** [comment]
Updated in 3 installations and works. Thank you Kamil for fixing!
